addons/vagrant/inventory/hosts

```diff
@@ -143,18 +143,19 @@ all:
       - name: 'mgmt'
         subnet: '172.17.17.0/24'
         forward_mode: 'route'
+        netmask: 255.255.255.0
 
       - name: 'registration'
         subnet: '172.17.2.0/24'
         forward_mode: 'route'
+        netmask: 255.255.255.0
 
       - name: 'isolation'
         subnet: '172.17.3.0/24'
         forward_mode: 'route'
+        netmask: 255.255.255.0
 
       - name: 'inline'
         subnet: '172.17.6.0/24'
         forward_mode: 'route'
-
-
-      
+        netmask: 255.255.255.0
```

Comment:
Yes but it is used when you set a specific ip. It is what I used for reg and iso network.
I guess we can let them here even if we remove iso and reg interface in wireless01.

Reply:
OK, I understand. It should be possible to set up an IP using X.X.X.X/24 with `ip`

Reply:
Hum, Not sure it will work. For example: X.X.X.X/24 with `ip` is not used like that in https://github.com/inverse-inc/packetfence/blob/devel/addons/vagrant/pfservers/Vagrantfile#L33